controls/SV-251703.rb

```diff
@@ -25,11 +27,12 @@
 Add or modify the following line:
 #includedir /etc/sudoers.d'
   impact 0.5
+  ref 'DPMS Target Red Hat Enterprise Linux 7'
   tag severity: 'medium'
   tag gtitle: 'SRG-OS-000480-GPOS-00227'
-  tag satisfies: nil
+  tag 'satisfies'
```

Comment:
Should I manually revert this to the nil value to retain the original code? @em-c-rod